update tiers docs for membership APIs (#45)
team-plain/docs: api-reference/graphql/tiers/add-members.mdx

You can add multiple tenants and companies to a tier in a single mutation.

> An entity (company or tenant) can only be part of a single tier.

Reviewer: ```suggestion
Companies and tenants can only be in a single tier.
```
